src/content/guides/libraries/create-packages.md

```diff
@@ -59,9 +54,10 @@ In most cases, each class should be in its own mini library, unless
 you have a situation where two classes are tightly coupled.
 
 :::note
-You may have heard of the `part` directive, which allows
-you to split a library into multiple Dart files. We recommend
-that you avoid using `part` and create mini libraries instead.
+You might know about the `part` directive.
+This directive allows you to split a library into multiple Dart files.
+Some might use part files to incorporate generated code into a library,
+but we recommend creating small libraries rather than using `part`.
```

Comment:
Can we stick with the text Bob originally [suggested](https://github.com/dart-lang/site-www/issues/2371#issuecomment-620261086)? Saying "Some might..." makes it sound like a personal choice rather than the actual "this is the only thing you should ever use them for"

```suggestion
Part files are useful for incorporating generated code into a library,
but we otherwise recommend that you don't use them.
Instead, create small libraries.
```
Or something like that.